server/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG KEYCLOAK_DIST=https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloa
 
 USER root
 
-RUN yum update -y && yum install -y epel-release git && yum install -y jq openssl which && yum clean all
+RUN microdnf update -y && microdnf install -y gzip hostname java-11-openjdk-headless openssl tar which && microdnf clean all
```

Comment:
@douglaspalmer we might need to switch to JDK 8 here. Unfortunately, that's not your fault but things will fall apart while building Keycloak with JDK 11.

Reply:
The ideal solution is to fix the build with JDK 11, but I'm not sure about the plans around it. @stianst any thoughts on it?

Reply:
Fixing build for JDK 11 should be done, so I'd rather get that fixed than to introduce JDK8 in the container.

Reply:
@stianst agreed on that, we do have a Jira for it https://issues.jboss.org/browse/KEYCLOAK-11185. Actually the build fails for JDK9 and beyond and here comes my question. Should we target JDK11 and not worry about JDK8, 9 and 10. Or it is mandatory that the build works with all these JDKs?

Reply:
I'd say let's have that conversation elsewhere ;)

For now it would be good to get KEYCLOAK-11185 fixed asap so we can merge this.